lib/ClangImporter/ImportDecl.cpp

```diff
@@ -525,6 +525,7 @@ static AccessorDecl *makeEnumRawValueGetter(ClangImporter::Implementation &Impl,
                      TypeLoc::withoutLoc(rawTy), enumDecl);
   getterDecl->setImplicit();
   getterDecl->setIsObjC(false);
+  getterDecl->setIsDynamic(false);
```

Comment:
I'm not happy with this proliferation of "things we have to set explicitly when synthesizing a decl". How is anyone supposed to remember the full list?

Reply:
I'm not happy either, but I think we should surface the duplication and then think of a way to eliminate it.